res/app/components/stf/device/enhance-device/enhance-device-service.js

```diff
@@ -45,7 +45,8 @@ module.exports = function EnhanceDeviceServiceFactory($filter, AppState) {
   }
 
   function enhanceDevice(device) {
-    device.enhancedName = device.name || device.model || device.serial || 'Unknown'
+    device.enhancedName = device.marketName || device.name || device.model || device.serial
```

Comment:
imho device name is the most unique setting that use can set to differentiate the same marken/model devices, could we bring device name first on list back?